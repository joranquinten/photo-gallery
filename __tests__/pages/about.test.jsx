import React from 'react';
import { shallow } from 'enzyme';

import About from '../../pages/about';

describe('About', () => {
  const mockContent = {
    id: 'id',
    title: 'title',
  };

  it('should render without crashing', () => {
    expect(shallow(<About content={mockContent} />)).toMatchSnapshot();
  });

  it('should render with contents provided', () => {
    expect(
      shallow(<About content={{ ...mockContent, contents: '<p>Contents</p>' }} />)
    ).toMatchSnapshot();
  });
});
