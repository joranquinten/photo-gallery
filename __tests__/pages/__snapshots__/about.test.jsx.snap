// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render with contents provided 1`] = `
<About
  content={
    Object {
      "contents": "<p>Contents</p>",
      "id": "id",
      "title": "title",
    }
  }
/>
`;

exports[`About should render without crashing 1`] = `
<About
  content={
    Object {
      "id": "id",
      "title": "title",
    }
  }
/>
`;
